### PR TITLE
Release/1.1.0

### PR DESCRIPTION
**Pull request by nanux**
* Artifacthub change annotations
* Bamboo 0.0.1 first release 📦 🥳 ⛵ - not officially supported but available for internal testing. I didn't produce changelog for it as it is the first cut.
* Changelog.md updates
* Update version to 1.1.0 for the officially supported products + Crowd
* Added support for `-DrecordOutput=true` system property to override the expected file to save some manual work
* Updated release documentation accordingly (internal doc)

### CODE REVIEW

**Review by errcode1202 — APPROVED** [review]

File: src/main/charts/jira/Changelog.md [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* DCKUBE-721: Update version in Chart.yaml files 
+* DCKUBE-733: Update the product versions (#345) 
+* DCD-1380: Update Jira to 8.20 LTS (#339) 
+* DCKUBE-739: Fix typos (#337) 
+* DCKUBE-739: make securityContext changes backward compatible (#332) 
+* Roll Statefulset Pods if ConfigMap changes (#315) 
+* Dckube 677 make security context more flexible (#321)
```
Comment (nghazali) [inline src/main/charts/jira/Changelog.md:18]: change to `DCKUBE-667: Make ...`

File: src/main/charts/jira/Changelog.md [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+
+* DCKUBE-721: Update version in Chart.yaml files 
+* DCKUBE-733: Update the product versions (#345) 
+* DCD-1380: Update Jira to 8.20 LTS (#339) 
+* DCKUBE-739: Fix typos (#337) 
+* DCKUBE-739: make securityContext changes backward compatible (#332) 
+* Roll Statefulset Pods if ConfigMap changes (#315)
```
Comment (nghazali) [inline src/main/charts/jira/Changelog.md:17]: To be consistent can we have the ticket number for this change as well?

**Review by nghazali — CHANGES_REQUESTED** [review]

File: src/main/charts/jira/Changelog.md (hunk 2 above)
Comment (nanux) [reply src/main/charts/jira/Changelog.md:17]: There was no ticket, as this was a community contribution

File: src/main/charts/jira/Changelog.md (hunk 1 above)
Comment (nanux) [reply src/main/charts/jira/Changelog.md]: Done

**Review by nghazali — APPROVED** [review]

**Pull request merged**